src/agent/communicator/src/http_client.cpp

```diff
@@ -231,13 +226,18 @@ namespace http_client
         boost::beast::detail::base64::encode(&basicAuth[0], userPass.c_str(), userPass.size());
 
         const auto reqParams = http_client::HttpRequestParams(
-            boost::beast::http::verb::post, host, port, "/security/user/authenticate", useHttps, "", basicAuth);
+            boost::beast::http::verb::post, host, "/security/user/authenticate", "", basicAuth);
 
         const auto res = PerformHttpRequest(reqParams);
 
         if (res.result() != boost::beast::http::status::ok)
         {
             LogError("Error: {}.", res.result_int());
+            LogError("Host: {}, Port: {}, Endpoint: {}, Https: {}",
```

Comment:
Why do you need this log? Why it is only here?

Reply:
PerformHttpRequest is failing on non https requests. I needed to check what the parameters were. I actually expected to have some feedback from you on whether this log was ok to keep. Having just the result is not that useful.

Reply:
I would say to remove it. We can analyze what logs can we improve later.

Reply:
Removed